test/run.sh

```diff
@@ -3,4 +3,4 @@ if [ "$#" -ne 1 ] || ! [ -r "$1" ]; then
   echo "Usage: $0 <config.el>" >&2
   exit 1
 fi
-/usr/bin/emacs -q --no-site-file --no-site-lisp --no-splash -l "$1"
+/usr/bin/env emacs -q --no-site-file --no-site-lisp --no-splash -l "$1"
```

Comment:
This does not work for me since I overwrite emacs in .local/bin, but I don't really care about this. Use env if you prefer that, since it is a bit more portable.

Reply:
I changed it since it did not work for me :laughing:

Reply:
What about just `emacs` would that work for you, too?

Reply:
No, but use what works for most people :)

Reply:
But probably I should my emacs wrapper script such that it recognizes if called with -Q or -q and then does not add its magic. Then /usr/bin/env emacs, or just emacs would work.